### DISCUSSION TITLE
Bun docs "Change port" unclear

### DISCUSSION THREAD

**Discussion by dominictobias** [honojs/hono#1764] [Q&A] [answered]
Hi, regarding this section: https://hono.dev/getting-started/bun#change-port-number

It's unclear what file should export this. The example project has a single file which exports `const app = new Hono()` as default already.

**Comment by yusukebe** [comment]
cc: @sor4chi

**Comment by paniaguaadrian** [comment, marked as answer]
Hi there! You should write it like this: 
`import { Hono } from 'hono';

const app = new Hono();

app.get('/', (c) => {
  return c.text('Hello Hono!');
});

export default {
  port: 4001,
  fetch: app.fetch,
};
`

**Comment by sor4chi** [comment]
https://github.com/honojs/website/pull/215
I refactored the documentation to make it more understandable.
